src/components/checkout.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+
+export default function Checkout({ selectedBirds, updateSelectedBirds }) {
```

Comment:
I highly recommend refactoring this code to use one slice of state, and one more generic function to update the state tied to each HTML component. Let me know if you need some more pointers on how to do it, but I think it's an amazing exercise to try and think of yourself - One state, One function